src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -141,15 +141,15 @@ public final class JavadocTokenTypes {
      * <pre>{@code @serialField counter Integer objects counter}</pre>
      * <b>Tree:</b>
      * <pre>{@code
-     *   |--JAVADOC_TAG[3x0] : [@serialField counter Integer objects counter]
-     *       |--SERIAL_FIELD_LITERAL[3x0] : [@serialField]
-     *       |--WS[3x12] : [ ]
-     *       |--FIELD_NAME[3x13] : [counter]
-     *       |--WS[3x20] : [ ]
-     *       |--FIELD_TYPE[3x21] : [Integer]
-     *       |--WS[3x28] : [ ]
-     *       |--DESCRIPTION[3x29] : [objects counter]
-     *           |--TEXT[3x29] : [objects counter]
+     * JAVADOC_TAG -> JAVADOC_TAG
+     *  |--SERIAL_FIELD_LITERAL -> @serialField
+     *  |--WS ->
+     *  |--FIELD_NAME -> counter
+     *  |--WS ->
+     *  |--FIELD_TYPE -> Integer
+     *  |--WS ->
+     *  |--DESCRIPTION -> objects counter
+     *      |--TEXT -> objects counter
```

Comment:
As sir suggested to refer the ```PARAM_LITERAL``` I referred [this PR](https://github.com/checkstyle/checkstyle/pull/14637)  and [this Javadoc](https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#PARAM_LITERAL)

Reply:
@Prathamesh-007 , what they mean is you are manually editing the printed AST which is not ideal.
Please only copy-paste the part that the maintainers have suggested.

From what I see, you've changed the format of the AST during manual removals

Reply:
Oh ok, so you mean I should copy/paste the output part that was generated using ```java -jar checkstyle-10.14.2-all.jar -J Test.java | sed "s/[[0-9]+:[0-9]+]//g"``` command instead of manually editing the format?

Reply:
> copy/paste the output part that was generated

only the part suggested, yes